README.md

```diff
@@ -4,6 +4,20 @@
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
 > This repository does not pretend to give you all answers for LinkedIn questions, it just reflects my real life experience.. If you guys have found some incorrect answers, or want to contribute your answers, please feel free to create PR, any changes are welcome!!!
 
+
+  
+<body>
+<table>
+  <tr>
+    <th> PROMOTE YOURSELF AND PUT YOUR ADD HERE !
+  </tr>
+  <tr>
+    <th> SEND US A MESSAGE FOR DETAILS!
```

Comment:
Again a very good idea @monkrus ! but would recommend you take these responsibilities on your own, so I put your email there!

```suggestion
    <th>SEND US AN <a href = "mailto: sergeisqa@gmail.com">EMAIL </a>FOR DETAILS!
```